### PR TITLE
Fix Vendetta on 193.4 (193204)

### PR DESCRIPTION
**Pull request by amsryq**
- Fixes the issue of Vendetta failing to load on Alpha 193204.
- **Does not** resolve the crashing "You tab" issue since the past builds (192204)

### CODE REVIEW

File: src/ui/components/index.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { ReactNative as RN } from "@metro/common";
-import { findByName, findByProps } from "@metro/filters";
+import { findByDisplayName, findByName, findByProps } from "@metro/filters";
 
 // Discord
 export const Forms = findByProps("Form", "FormSection");
 export const General = findByProps("Button", "Text", "View");
-export const Alert = findByProps("alertDarkStyles", "alertLightStyles").default;
+export const Alert = findByDisplayName("FluxContainer(Alert)");
```
Comment (Gabe616) [inline src/ui/components/index.ts:7]:
```suggestion
export const Alert = findByName("FluxContainer");
```

File: src/ui/components/index.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import { ReactNative as RN } from "@metro/common";
-import { findByName, findByProps } from "@metro/filters";
+import { findByDisplayName, findByName, findByProps } from "@metro/filters";
```
Comment (Gabe616) [inline src/ui/components/index.ts:2]:
```suggestion
import { findByName, findByProps } from "@metro/filters";
```

**Review by nexpid — CHANGES_REQUESTED** [review]
why not do this?
nvm, turns out there's multiple FluxContainers

**Pull request merged**